nemo/utils/model_copy.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
I don't know why, but it might be better to name the python script and bash script differently haha - i had to double take.